playground/nextjs-app-router/components/demo/Swap.tsx

```diff
@@ -64,52 +62,39 @@ function SwapComponent() {
 
   return (
     <div className="relative h-full w-full">
-      {address ? (
-        chainId !== 8453 ? (
-          <div className="absolute top-0 left-0 z-10 flex h-full w-full flex-col justify-center rounded-xl bg-[#000000] bg-opacity-50 text-center">
-            <div className="mx-auto w-2/3 rounded-md bg-muted p-6 text-sm">
-              Swap Demo is only available on Base.
-              <br />
-              Please change your chain in settings.
-            </div>
-          </div>
-        ) : (
-          <></>
-        )
-      ) : (
+      {chainId !== 8453 ? (
         <div className="absolute top-0 left-0 z-10 flex h-full w-full flex-col justify-center rounded-xl bg-[#000000] bg-opacity-50 text-center">
           <div className="mx-auto w-2/3 rounded-md bg-muted p-6 text-sm">
-            Swap Demo requires wallet.
+            Swap Demo is only available on Base.
             <br />
-            Please connect in settings.
+            Please change your chain in settings.
```

Comment:
Maybe something like:
```suggestion
You're connected to a different network. Switch to Base and continue using the app.
```

Reply:
I'd also add some copy about the Swap being disabled on Playground and for them to run it locally if they want to test swaps